resources/assets/js/components/RowServiceAdmin.vue

```diff
@@ -7,13 +7,13 @@
           <a v-if="s.source && s.source.toUpperCase() === 'vesta'.toUpperCase()" :href="url" target="_blank">{{ s.source.toUpperCase() }}</a>
         <span v-else>{{ s.source || '' }}</span>
       </td>
-      <td :class="statusClass">
+      <td :class="getStatusClass(s.status)">
         <span
-          :data-toggle="statusTooltip ? 'tooltip' : null"
-          :title="statusTooltip"
+          :data-toggle="getStatusTooltip(s.status) ? 'tooltip' : null"
```

Comment:
```suggestion
          :data-toggle="getStatusTooltip(s.status) || 'tooltip'"
```

Reply:
getStatusTooltip(s.status) doesn't return 'tooltip' unfortunately.
Did revisit that function and fixed it so the switch returns null as default.